### PR TITLE
update ember-data to 3.28.13

### PR DESCRIPTION
**Pull request by lagartoverde**
Seems to solve the problems with the participation list, https://binnenland.atlassian.net/browse/GN-4118

### CODE REVIEW

**Review by elpoelma — CHANGES_REQUESTED** [review]
Secretaries of a meeting are not correctly fetched. The following error is displayed: 
image

**Comment by nvdk** [comment]
I think it would be better to await the functionaris higher up in the power select or make sure it's loaded in loadData

File: app/models/functionaris.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import Model, { attr, belongsTo } from '@ember-data/model';
+import MandatarisModel from './mandataris';
 
-export default class FunctionarisModel extends Model {
+export default class FunctionarisModel extends MandatarisModel {
```
Comment (abeforgit) [inline app/models/functionaris.js:4]: why did you do this? and don't we need to set polymorphic on some relationships then?

File: app/components/participation-list.hbs [hunk 2]
```diff
@@ -11,7 +11,7 @@
       {{/unless}}
     </li>
     <li class="au-c-list-divider__item">
-      {{t "participation-list.secretaris-label"}} <strong>{{this.secretary.isBestuurlijkeAliasVan.fullName}}</strong>
+      {{t "participation-list.secretaris-label"}} <strong></strong>
```
Comment (abeforgit) [inline app/components/participation-list.hbs:14]: theres an empty strong tag here

File: package.json [hunk 3]
```diff
@@ -66,7 +66,7 @@
     "ember-concurrency": "^2.3.7",
     "ember-config-helper": "^0.1.3",
     "ember-could-get-used-to-this": "^1.0.1",
-    "ember-data": "~3.27.1",
+    "ember-data": "~3.28.12",
```
Comment (abeforgit) [inline package.json:69]: could you bump to 3.28.13 while you're at it

**Review by abeforgit — CHANGES_REQUESTED** [review]

**Comment by lagartoverde** [comment]
@elpoelma is this finished? Because it looks good to me, and it works, so if it is, you can merge

**Comment by elpoelma** [comment]
> I think it would be better to await the functionaris higher up in the power select or make sure it's loaded in loadData

I agree, I will try to look into this in this PR, or a future PR.

**Comment by elpoelma** [comment]
> > I think it would be better to await the functionaris higher up in the power select or make sure it's loaded in loadData
> 
> I agree, I will try to look into this in this PR, or a future PR.

I honestly also don't see why the `await` would be needed, as the `functionaris`-models are already resolved in the `loadData` function.

**Review by abeforgit — CHANGES_REQUESTED** [review]
- some problem with inverses when loading the people present at the meeting
image

kind of breaks the whole meeting page

**Comment by elpoelma** [comment]
> * some problem with inverses when loading the people present at the meeting
>       image
> 
> 
> kind of breaks the whole meeting page

should be resolved now, see https://github.com/lblod/frontend-gelinkt-notuleren/pull/462#issuecomment-1562407050

**Review by abeforgit — CHANGES_REQUESTED** [review]
getting this error when trying to open an agenda publication preview:
image

**Pull request merged**